zips/zip-0234.md

```diff
@@ -141,21 +141,21 @@ testing the "less than" conditional with every block after the second halving.
 We prefer the pre-defined constant height parameter, to give everyone more
 _time_ certainty at the cost of _issuance level_ certainty.
 
-The difference in up-front calculation versus dynamic calculation is in
-whether or not burns are accounted for (since future burns cannot be
-calculated up-front). This means with the pre-defined constant parameter
-approach, issuance will jump _up_ some amount at activation. This amount
-should be equivalent to all ZEC burnt prior to that height times
-$\mathsf{BLOCK\_SUBSIDY\_FRACTION}$. For example, if a total of 100,000 ZEC
-were burnt prior to the pre-defined constant activation height, then at
-activation the issuance would be larger than BTC-style issuance by
-$100\_000\textsf{ ZEC} \cdot \mathsf{BLOCK\_SUBSIDY\_FRACTION}$,
-which we calculate equals $0.04126$ ZEC. This example is chosen to
-demonstrate that a very large burn amount (much larger than expected) would
-elevate issuance by a relatively small amount. For this reason, we believe
-a pre-defined constant is a better approach to achieving Key Objective 6
-than a "dynamic latch" logic because it is so much simpler to implement
-and reason about.
+The difference in up-front calculation versus dynamic calculation is
+in whether or not funds removed from circulation are accounted for
+(since funds removed from circulation in the future cannot be calculated
+up-front). This means with the pre-defined constant parameter approach,
+issuance will jump _up_ some amount at activation. This amount should be
+equivalent to all ZEC removed from circulation prior to that height times
+$\mathsf{BLOCK\_SUBSIDY\_FRACTION}$. For example, if a total of 100,000 ZEC were
+burnt prior to the pre-defined constant activation height, then at activation
+the issuance would be larger than BTC-style issuance by $100\_000\textsf{ ZEC}
+\cdot \mathsf{BLOCK\_SUBSIDY\_FRACTION}$, which we calculate equals $0.04126$
+ZEC. This example is chosen to demonstrate that a very large amount removed from
+circulation (much larger than expected) would elevate issuance by a relatively
+small amount. For this reason, we believe a pre-defined constant is a better
+approach to achieving Key Objective 6 than a "dynamic latch" logic because it is
+so much simpler to implement and reason about.
 
 $\mathsf{MoneyReserveAfter}(\mathsf{height}) =$ The value of the Money Reserve
```

Comment:
This should still be part of the Specification; it should be moved above the two preceding paragraphs after they are subsectioned per the previous comment.